### ISSUE TITLE
[Low] Fallback to English WCAG terms when untranslated

### ISSUE THREAD

**Issue by remibetin** [w3c/wai-wcag-em-report-tool#139]
Updating the tool for WCAG 2.2. meant adding new SCs in `wcag.json` translated files (Dutch & French versions) for the tool to continue to work in the translated language.

Issues:
- When new SCs are added, we have to manually add them, untranslated, in all existing translations.
- Internationalization issue: these untranslated titles and paragraphes are not declared using lang="en". (see https://www.w3.org/International/questions/qa-html-language-declarations)

Possible solutions:
- Fallback to English version when there is no translation for WCAG terms (it seems to work like that for UI terms)
- Declare `lang="en"` to elements surrounding that content

**Comment by remibetin** [comment]
@mhansma96 Would you know where the current fallback to English for non-translated UI terms is managed in the code?

**Comment by mhansma96** [comment]
@remibetin You might get closer to what you're looking for either here: https://github.com/w3c/wai-wcag-em-report-tool/tree/main/src/locales/en (the English translation folder to set them)

Or here: https://github.com/w3c/wai-wcag-em-report-tool/blob/main/src/components/App.svelte (This is where the context of the app is set and languages are imported)
In addition, one of the translate objects comes from here: https://github.com/w3c/wai-wcag-em-report-tool/blob/main/src/stores/i18nStore.js

**Comment by rvantonisse** [comment]
Hey, a quick tldr answer: default locale is first locale entered at src/locales/index.json

The file / locales are passed through to i18n initializing script src/scripts/i18n.js and there the first locale is picked as default.

**Comment by remibetin** [comment]
Thank you very much @mhansma96 & @rvantonisse for your answers. That seems to work for UI terms translations, but not for WCAG guidelines/SC translations though.

**Comment by rvantonisse** [comment]
Hey @remibetin ,

WCAG does not have fallback translations due to different translation call (I think...) 

At src/components/ui/auditor/auditorview.svelte the translations are called on principles and guidelines as json object. In same folder criterion.svelte criterion translations are pulled as object.

This is what I think is the cause. If there is no translation for WCAG2.2 criteria then this will be missing in the criterion json objects.

I would expect the translation plugin used would pull the missing object from the default locale.

There are 2 translation calls, you can find them in src/stores/i18nStore.js. The first works for UI, the second on WCAG.

You may try to call wcag translations with the translate function as normal UI. This looked not possible at the time.

**Comment by mhansma96** [comment]
You might also find some traces of the WCAG here: https://github.com/w3c/wai-wcag-em-report-tool/blob/main/src/locales/en/WCAG.json

And here: https://github.com/w3c/wai-wcag-em-report-tool/blob/main/src/stores/wcagStore.js
